libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryUtils.java

```diff
@@ -121,20 +119,18 @@ public static String obtainUniversalUniqueIdentifier() {
   }
 
   public static String obtainApplicationState(Context context) {
-    ActivityManager activityManager = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
```

Comment:
Shall we also remove the permission from the AndroidManifest.xml?
```
    <uses-permission android:name="android.permission.GET_TASKS" android:maxSdkVersion="20"/>
```

Reply:
Sure!